apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.tsx

```diff
@@ -110,43 +109,87 @@ router.get(
 router.post(
   '/',
   asyncHandler(async (req, res) => {
+    const { course } = extractPageContext(res.locals, {
+      pageType: 'course',
+      accessType: 'instructor',
+    });
+
     if (req.body.__action === 'add_course_instance') {
-      if (!req.body.short_name) {
-        throw new error.HttpStatusError(400, 'short_name is required');
+      const { short_name, long_name, start_date, end_date } = z
+        .object({
+          short_name: z.string(),
+          long_name: z.string(),
```

Comment:
Let's trim `short_name` and `long_name`. In general we should be trimming almost all text inputs.

Reply:
Fixed in ff72e98b663af4d90c1a02b80b80ed59b794cfe7 and d46a62b4c6d6c1c5f43be7fe5cdf4f91ab35817e.